addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -780,6 +787,10 @@
             </div>
 
             <div t-if="channel.can_upload" class="text-end pb-2 col-auto">
+                <!-- TODO: In master, remove the aria-label of the next two
+                    anchors, as they already have a title, to avoid an
+                    accessibility error of type `Elements with visible text
+                    labels do not have matching accessible names`. -->
```

Comment:
1. IMO a TODO never done is better than no TODO as it at least advertises that the following code is bad for the next person's to read. And if the next person is me, I'll definitely change it.
2. This TODO will not reach master, the forward-port PR will be push-forced to adapt those directly. That's actually the main point of this TODO "we wanted to do that but since we chose to target 17.0, we'll delay that for the forward-port".